eng/targets/GenerateServiceHubConfigurationFiles.targets

```diff
@@ -13,6 +13,7 @@
     <ItemGroup>
       <_ServicesWithBitness Include="@(ServiceHubService)" FileSuffix="" HostSuffix=".x86" HostIdSuffix="32" />
       <_ServicesWithBitness Include="@(ServiceHubService)" FileSuffix="64" HostSuffix="" HostIdSuffix="" />
+      <_ServicesWithBitness Include="@(ServiceHubService)" FileSuffix="64S" HostSuffix="" HostIdSuffix="S" />
```

Comment:
wha'ts the reasoning behind the S suffix?  'Server'?  Could we just do 'Server' maybe for clarity?  i can see the S being easily missed.

Reply:
The suffix is temporary and not intended to suggest specific meaning to viewers. After the A/B work we'll remove this service and update the non-suffix one to have the winning behavior.

Reply:
Are we adopting the name scheme for Xbox? I will name .net core version 64 seriex X then ;)